prototype/mc_shortest_path/arduino_mc_shortest_path/mc_shortest_path.py

```diff
@@ -0,0 +1,32 @@
+import struct
+import time
+import serial
+import numpy as np
+
+def main():
+    np.set_printoptions(suppress=True) # Prevents from printing in scientific notation
+
+    port = serial.Serial('/dev/tty.usbmodem2101', 9600)
+    time.sleep(2)
```

Comment:
I think it would be usefull to explain why we need to sleep. Say something like, arduino takes time to initialize the serial connection